### ISSUE TITLE
Code for PEFT-LoRA tuning of bigscience/T0_3B on raft/twitter_complaints

### ISSUE THREAD

**Issue by VukVukovic** [huggingface/peft#226]
**Use Cases** section od README mentions PEFT-LoRA tuning of [bigscience/T0_3B](https://huggingface.co/bigscience/T0_3B) on [ought/raft/twitter_complaints](https://huggingface.co/datasets/ought/raft/viewer/twitter_complaints) achieving the accuracy of 0.863. Is there a notebook/code available?

**Comment by pacman100** [comment]
Please check the sample submission that I made on raft leaderboard (21st rank at present) with LoRA achieving  accuracy of 0.863. There was no hyperparam tuning done. Let me find the notebook that I used and will create a new repo with notebooks for raft datasets. This will take couple weeks.

<img width="1364" alt="Screenshot 2023-03-29 at 7 22 34 PM" src="https://user-images.githubusercontent.com/13534540/228560227-abd7f679-51f1-4fb3-b8da-9fb81b839f0a.png">

**Comment by VukVukovic** [comment]
@pacman100, thank you very much for getting back to me with the pointers! I saw the submission result on the RAFT leaderboard with the matching accuracy, however I was looking for the code (even if not fully prepared for the public repo).

To be more precise, I am trying to perform prompt tuning of [bigscience/T0_3B](https://huggingface.co/bigscience/T0_3B) on [ought/raft/twitter_complaints](https://huggingface.co/datasets/ought/raft/viewer/twitter_complaints). I created the code following your [example](https://github.com/huggingface/peft/blob/main/examples/causal_language_modeling/peft_prompt_tuning_clm.ipynb) and some [T5 fine-tuning example](https://github.com/huggingface/notebooks/blob/main/examples/summarization.ipynb). However, I am not getting the expect output (e.g. `a not is is is`, `a not the`, `a  is complaint`). I was using `SEQ_2_SEQ_LM` as a PromptTuning task type and `complaint`, `no complaint` as labels.

**Comment by VukVukovic** [comment]
@pacman100 Could you please give some heads up about the approach you used? Did you use `complaint`, `no complaint` as labels? Were you considering this as a seq2seq problem (and thus used `SEQ_2_SEQ_LM` task type)?
